Laba2SCCS/src/BusinessLayer/Services/DriverService.cs

```diff
@@ -0,0 +1,146 @@
+﻿using AutoMapper;
+using BusinessLayer.Entities;
+using BusinessLayer.Infrastructe;
+using BusinessLayer.Interfaces;
+using DataAccessLayer.DTO;
+using DataAccessLayer.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Services
+{
+    internal class DriverService : IService<Driver>
+    {
+        private readonly IGenericRepository<DriverDto> _driverRepository;
+        private readonly IMapper _mapper;
+
+        public DriverService(IGenericRepository<DriverDto> driverRepository)
+        {
+            _driverRepository = driverRepository;
+
+            _mapper = new MapperConfiguration(cfg =>
```

Comment:
Please, create a single mapping profile in a separate file in BLL and register it in Program.cs